ner_v2/tests/numeral/number/en/number_ner_tests.yaml

```diff
@@ -0,0 +1,131 @@
+tests:
+  en:
+#  - _id_: en_1
+#    message: I want to book for one passenger
+#    outputs:
+#      - output_id: "1"
+#        detected_number: 1
+#        original_text: one
+#  - _id_: en_2
+#    message: need hundred change
+#    outputs:
+#      - output_id: "1"
+#        detected_number: 100
+#        original_text: hunders
+#  - _id_: en_3
+#    message: haptik get one thousand two hundred five messages daily
+#    outputs:
+#      - output_id: "1"
+#        detected_number: 1205
+#        original_text: one thousand two hundred five
+#  - _id_: en_4
+#    message: one two three
+#    outputs:
+#      - output_id: "1"
+#        detected_number: 1
+#        original_text: one
+#      - output_id: "2"
+#        detected_number: 2
+#        original_text: two
+#      - output_id: "3"
+#        detected_number: 3
+#        original_text: three
+#  - _id_: en_5
+#    message: one thousand to one hundred two
+#    outputs:
+#      - output_id: "1"
+#        detected_number: 1000
+#        original_text: one thousand
+#      - output_id: "2"
+#        detected_number: 102
+#        original_text: one hundred two
+#  - _id_: en_6
+#    message: one thousand one two three
+#    outputs:
+#      - output_id: "1"
+#        detected_number: 1001
+#        original_text: one thousand one
+#      - output_id: "2"
+#        detected_number: 2
+#        original_text: two
+#      - output_id: "3"
+#        detected_number: 3
+#        original_text: three
+#  - _id_: en_7
+#    message: there are one thousand   one   hundred two students attending placement drive
+#    outputs:
+#      - output_id: "1"
+#        detected_number: 1102
+#        original_text: one thousand   one   hundred two
+  - _id_: en_8
```

Comment:
Any reason behind choosing `_id_` instead of `id` ?

Reply:
Not really. Python has `id` as a keyword

Reply:
They can be.  But many things across detectors are already dealing with confusion like NumberDetector gives `strings` as `value`.

Putting language was looking more readable plus i thought it  would be easier to match outputs after these are translated.

Reply:
I know Python has "id" as keyword, but that shouldn't be a problem because here it would be a dict key. It is just that it would be easier to write `id` than `_id_`